EXAMPLES.md

```diff
@@ -292,15 +292,17 @@ ShopifyBuy.Client().products((products, error) => {
 });
 ```
 
-## Apple Pay checkout
+## Native pay checkout
 
-You can allow users to pay with Apple Pay, providing a seamless checkout experience.
+You can allow users to pay with native pay (Apple Pay or Android Pay, depending on the device OS), providing a seamless checkout experience.
 
-To determine whether the user is able to make a payment with Apple Pay, you can use the `CanCheckoutWithNativePay` method. If the user has this capability, then you can use `CheckoutWithNativePay` to present the Apple Pay authentication interface to the user. If they do not, then you can accept payment using [Web View Checkout](#web-view-checkout).
+To determine whether the user is able to make a payment with native pay, you can use the `CanCheckoutWithNativePay` method. If the user has this capability, then you can use `CheckoutWithNativePay` to present the native pay authentication interface to the user. If they do not, then you can accept payment using [Web View Checkout](#web-view-checkout).
 
 `CheckoutWithNativePay` takes in 4 parameters:
 
-* `key` is the Merchant ID of your application found on your [Apple Developer Portal](https://developer.apple.com/library/content/ApplePay_Guide/Configuration.html)
+* `key`
+  * **On iOS** is the Merchant ID of your application found on your [Apple Developer Portal](https://developer.apple.com/library/content/ApplePay_Guide/Configuration.html).
+  * **On Android** is a Base64-encoded public key found on _Admin_ > _Apps_ > _Manage private apps_ > _{Your private app}_ > _Storefront API_ > _Configure Mobile Buy SDK settings (optional)_ > _MOBILE BUY SDK (ANDROID)_.
```

Comment:
I would prefix the _Admin_->_Apps_ ... stuff saying that that is on the Shopify Admin site:

`**On Android** is a Base64-encoded public key that can be found on your Shopify Admin page. To locate the key, navigate to ...`

Reply:
https://github.com/Shopify/unity-buy-sdk/pull/362/commits/dde30a03aa1fc4bda959b69000b8a533647ebbbb